aemedge/styles/styles.css

```diff
@@ -764,7 +796,7 @@ margin:0 auto;
 /* stylelint-disable-next-line selector-class-pattern */
 .section.package-cards-container #package-cards-app .hlmNfh
 {
-  flex-direction: column;
+  flex-direction: row;
```

Comment:
You can delete this once you've deleted line 694, because the app's embedded style already has this.
![image](https://github.com/user-attachments/assets/1740bd1e-94e9-4a4f-ad0e-9f7e007772de)